Docker environment

EvanAntoni opened the pull request:
Added configuration for deployment on Docker. Also added add-on for docker compose 
for simplified management of application services.

EvanAntoni committed:
add dockerfile for backend and frontend

EvanAntoni committed:
add containers configuration for docker compose

EvanAntoni committed:
add script for services management

serge30 commented on Dockerfile:
```diff
@@ -0,0 +1,7 @@
+FROM python:2.7.15
```
`python:2.7` is enough.

serge30 commented on frontend/Dockerfile:
```diff
@@ -0,0 +1,11 @@
+FROM node:10.15.0
```
`node:10.15` is enough. Please try do not set hotfix version and use only parts. As there might be new hotfix releases which fixes some vulnerabilities and bugs, and you'd like to pick them.

serge30 reviewed (changes requested):
Please update `README.md` with steps to deploy the project on dev environments using your script and Docker.

EvanAntoni committed:
edit image version for node and python

EvanAntoni committed:
edit environment section in db service

EvanAntoni committed:
add database conection and server url for Docker environment

EvanAntoni committed:
edit help in "smanager" deployment script

EvanAntoni committed:
add deployment instruction to README.md

EvanAntoni commented:
- Fixed version of images
- Added deployment instructions through Docker into README.md

Additionally:
- Corrected help in the deployment script
- Made minor changes to the docker-compose.yml and development.ini
